src/app/pet.service.ts

```diff
@@ -13,24 +13,62 @@ import { PETS } from './mock-pets';
 export class PetService {
 
   private petsUrl = 'api/pets';
+  httpOptions = {
+    headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+  };
+
+  pets$ = this.http.get<Pet[]>(this.petsUrl)
```

Comment:
If this service is shared across many components, you should look into two ways to share observables across subscribers because it may not be doing what you think it's doing.

First approach is `publishReplay` and `refCount` That looks like this:
```typescript
    this.isSubmitted = this.isSubmittedStream.asObservable()
      .pipe(
        publishReplay(1),
        refCount()
      );
```

The second way is `shareReplay`
```typescript
    this.user = this.refreshStream.asObservable()
      .pipe(
        switchMap(() => this.getUserDetails(userId)),
        shareReplay(1)
      );
```

Research some of these for a later commit to see if you should be using them.